src/mapper/shelley.rs

```diff
@@ -228,6 +232,19 @@ impl EventWriter {
             self.append(EventData::BlockEnd(record))?;
         }
 
+        for witness in block.transaction_witness_sets.iter() {
```

Comment:
Maybe we would like to allow to distinguish witnesses scripts from auxiliary data ones?